tcms/testruns/views.py

```diff
@@ -460,7 +460,8 @@ def ajax_search(request, template_name='run/common/json_runs.txt'):
 
         # Get associated statistics data
         run_ids = [run.pk for run in searched_runs]
-        qs = TestCaseRun.objects.filter(run__in=run_ids).values('run').annotate(cases_count=Count('case'))
+        qs = TestCaseRun.objects.filter(run__in=run_ids)\
+            .values('run').annotate(cases_count=Count('case'))
```

Comment:
Elsewhere we use a different style for queries spanning multiple lines. For example:
```
qs = TestCaseRun.objects.filter(
    run__in=run_ids
).values(
    'run'
).annotate(
    cases_count=Count('case')
)
```

Please update it